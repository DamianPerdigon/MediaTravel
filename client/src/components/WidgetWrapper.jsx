// Importing Box component and styled utility from Material-UI
import { Box } from "@mui/material";
import { styled } from "@mui/system";

// Creating a styled component, WidgetWrapper, based on the Box component
const WidgetWrapper = styled(Box)(({ theme }) => ({
  // Setting padding for the widget
  padding: "1.5rem 1.5rem 0.75rem 1.5rem",

  // Using the theme's secondary light color for the background
  backgroundColor: theme.palette.secondary.light,
  borderRadius: "0.75rem",
}));

export default WidgetWrapper;
