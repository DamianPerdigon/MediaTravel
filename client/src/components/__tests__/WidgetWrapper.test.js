import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider } from '@mui/material/styles';
import WidgetWrapper from '../WidgetWrapper';
import { createTheme } from '@mui/material/styles';

const theme = createTheme(); // Crear un tema de MUI para el contexto del componente

describe('WidgetWrapper', () => {
  it('debe coincidir con el snapshot', () => {
    const { asFragment } = render(
      <ThemeProvider theme={theme}>
        <WidgetWrapper>
          {/* Contenido del WidgetWrapper si es necesario */}
        </WidgetWrapper>
      </ThemeProvider>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
