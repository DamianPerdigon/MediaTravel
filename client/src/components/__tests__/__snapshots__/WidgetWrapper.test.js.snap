// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetWrapper debe coincidir con el snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-zb59i3"
  />
</DocumentFragment>
`;
